homeassistant/components/calendar/services.yaml

```diff
@@ -18,6 +18,12 @@ create_event:
       example: "Meeting to provide technical review for 'Phoenix' design."
       selector:
         text:
+    location:
+      name: Location
+      description: The location of the event.
+      example: "Union Square, New York City"
```

Comment:
Optional comment: How about "Conference Room - F123, Bldg. 002" to be consistent with the other examples from the RFC? :)
https://www.rfc-editor.org/rfc/rfc5545#section-3.8.1.7

I think its kind of fun to keep to the rfc5545 examples for history, but not required

Reply:
Done.